assignments/prototype-refactor.js

```diff
@@ -2,39 +2,46 @@
 // Today your goal is to refactor all of this code to use ES6 Classes.
 // The console.log() statements should still return what is expected of them.
 
-function GameObject(options) {
-  this.createdAt = options.createdAt;
-  this.dimensions = options.dimensions;
-}
 
-GameObject.prototype.destroy = function() {
-  return `Object was removed from the game.`;
-};
 
-function CharacterStats(characterStatsOptions) {
-  GameObject.call(this, characterStatsOptions);
-  this.hp = characterStatsOptions.hp;
-  this.name = characterStatsOptions.name;
+// ES6 Version
+class GameObject{
+  constructor(attributes) {
+    this.createdAt = options.createdAt;
```

Comment:
It looks like you changed the constructor parameter to attributes but didn't change `options` to the right side of the assignment statements inside the constructor.